pages/index.js

```diff
@@ -1,22 +1,56 @@
 import React from "react";
-import Link from "next/link";
 import Footer from "../components/Footer/index.js";
-import Header from "../components/Header";
 import styled from "styled-components";
 
+const StyledHeader = styled.header`
+  background-color: #987554;
+  width: 100%;
+  padding: 20px;
+  text-align: center;
+`;
+
+const Logo = styled.p`
+  font-size: 24px;
+  color: black;
+`;
+
 const StyledHomePage = styled.main`
-  padding-bottom: 60px; /* Hier den gewünschten Abstand einstellen */
+  padding-bottom: 60px;
+  background-color: #e5d3b3;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
```

Comment:
i think in a next app you better go with `Link component` from next/link than the regular `anchor` tag. But its up to you. ✌🏼
```
const Button = styled(Link)`
display: flex;
.
.
.
`;
```